src/Data/Scripts/Script_002_InsertData.sql

```diff
@@ -0,0 +1,69 @@
+INSERT INTO "user" (id, name, email, passwordHash, createdAt) 
+VALUES
+('a1b2c3d4-e5f6-7g8h-9i0j-a1b2c3d4e5f6', 'John Doe', 'john.doe@example.com', 'hash1', '2023-09-01'),
```

Comment:
I got an invalid input sintax for uuid: `a1b2c3d4-e5f6-7g8h-9i0j-a1b2c3d4e5f6`
![image](https://github.com/user-attachments/assets/0f431bf5-53ec-4488-8888-f41d019b70d3)
![image](https://github.com/user-attachments/assets/4c2de0f0-b78b-4ab5-8b4b-66c33179c06b)